.prettierignore

```diff
@@ -7,7 +7,6 @@ coverage
 *.rs
 *.hbs
 target
-rust/
```

Comment:
This should be safe. I tested this out by adding boilerplate `*.ts` files into other rust directories and they did not get ran against. 

I think the `prettier` check won't check your entire folder path. Since I pass `./rust/kcl-language-server` we are safe. If I passed `./rust` then we would need to ignore other stuff.